syntax_and_semantics/cross-compilation.md

```diff
@@ -2,7 +2,20 @@
 
 Crystal supports a basic form of [cross compilation](http://en.wikipedia.org/wiki/Cross_compiler).
 
-In order to achieve this, the compiler executable provides two flags:
+## Preparation
+
+Cross-compilation will depend on `libcrystal` being present on the target system, so we need to build it by:
+* Downloading Crystal source code on the target machine.
```

Comment:
I'd suggest to use imperatives instead of gerunds. This better expresses intention and purpose.